docs/source/1.0/guides/building-models/build-config.rst

```diff
@@ -879,21 +879,23 @@ namespace. Shapes not connected to a service will not be flattened.
       - Description
     * - namespace
       - ``string``
-      - The target namespace.
+      - **REQUIRED** The target namespace.
     * - service
-      - ``string``
-      - The service to be flattened.
+      - ``shapeId``
+      - **REQUIRED** The service to be flattened.
     * - includeTagged
       - ``[string]``
       - The set of tags that, if found on a shape not connected to the service,
         forces the shape to have its namespace flattened into the target
-        namespace.
+        namespace. When additional shapes are included, the shapes are replaced
```

Comment:
This might need a minor clarification. It's not just the additional shapes that are replaced, right? It's any shape that's considered part of the closure of the provided `service`.

Reply:
Resolved in documentation on the service shapeId.